resources/js/components/open/forms/OpenFormField.vue

```diff
@@ -24,199 +33,253 @@
+      <component
+        :is="getFieldComponents"
+        v-if="getFieldComponents"
+        v-bind="inputProperties(field)"
+        :required="isFieldRequired"
+        :disabled="isFieldDisabled"
       />
       <template v-else>
-        <div v-if="field.type === 'nf-text' && field.content" :id="field.id" :key="field.id"
-              class="nf-text w-full px-2 mb-3" :class="[getFieldAlignClasses(field)]"
-              v-html="field.content"
+        <div
+          v-if="field.type === 'nf-text' && field.content"
+          :id="field.id"
+          :key="field.id"
+          class="nf-text w-full px-2 mb-3"
+          :class="[getFieldAlignClasses(field)]"
+          v-html="field.content"
         />
-        <div v-if="field.type === 'nf-code' && field.content" :id="field.id" :key="field.id"
-              class="nf-code w-full px-2 mb-3"
-              v-html="field.content"
+        <div
+          v-if="field.type === 'nf-code' && field.content"
+          :id="field.id"
+          :key="field.id"
+          class="nf-code w-full px-2 mb-3"
+          v-html="field.content"
         />
-        <div v-if="field.type === 'nf-divider'" :id="field.id" :key="field.id" 
-            class="border-b my-4 w-full mx-2"
+        <div
+          v-if="field.type === 'nf-divider'"
+          :id="field.id"
+          :key="field.id"
+          class="border-b my-4 w-full mx-2"
         />
-        <div v-if="field.type === 'nf-image' && (field.image_block || !isPublicFormPage)" :id="field.id"
-              :key="field.id" class="my-4 w-full px-2" :class="[getFieldAlignClasses(field)]"
+        <div
+          v-if="
+            field.type === 'nf-image' &&
+            (field.image_block || !isPublicFormPage)
+          "
+          :id="field.id"
+          :key="field.id"
+          class="my-4 w-full px-2"
+          :class="[getFieldAlignClasses(field)]"
         >
           <div v-if="!field.image_block" class="p-4 border border-dashed">
             Open <b>{{ field.name }}'s</b> block settings to upload image.
           </div>
-          <img v-else :alt="field.name" :src="field.image_block" class="max-w-full">
+          <img
+            v-else
+            :alt="field.name"
+            :src="field.image_block"
+            class="max-w-full"
+          />
         </div>
       </template>
+      <template v-else-if="field.type === 'phone_number'">
+        <phone-input v-model="dataFormValue[field.id]"></phone-input>
+      </template>
     </div>
   </div>
 </template>
 
 <script>
-import FormLogicPropertyResolver from '../../../forms/FormLogicPropertyResolver.js'
-import FormPendingSubmissionKey from '../../../mixins/forms/form-pending-submission-key.js'
+import FormLogicPropertyResolver from "../../../forms/FormLogicPropertyResolver.js";
+import FormPendingSubmissionKey from "../../../mixins/forms/form-pending-submission-key.js";
+import PhoneInput from "../components/forms/PhoneInput.vue";
```

Comment:
The goal of globally (lazily) importing the component in `index.js` is to avoid having to load it here, so you can get rid of this and just use the component straight away